modules/source-bookshelf/lib/process_sort.js

```diff
@@ -3,11 +3,12 @@ const _ = require('lodash');
 // TODO: investigate how to deal with express's query parser
 // converting + into a space.
 function isAscending (key) {
-  return key[0] === '+' || key[0] === ' ';
+  return key[0] === '+' || key[0] === '-';
 }
 
 module.exports = function (model, query, sortBy) {
   var validFields = model.fields;
+  console.log(model.fields);
```

Comment:
There is no such thing as model.fields. The example app has model.columns. If I hard-code this value in, the test below passes.